### PR TITLE
Copyright checks in notebook and script files

### PR DESCRIPTION
**Pull request by mingxin-zheng**
Signed-off-by: Mingxin Zheng <18563433+mingxin-zheng@users.noreply.github.com>

Adds the copyright check mentioned #1143 .

### Description
A few sentences describing the changes proposed in this pull request.

### Checks

- [x] Avoid including large-size files in the PR.
- [x] Clean up long text outputs from code cells in the notebook.
- [x] Remove private info such as user name and private key from the notebook and script files.
- [x] Ensure (1) hyperlinks and markdown anchors are working (2) use relative paths for tutorial repo files (3) put figure and graphs in the `./figure` folder
- [x] Notebook runs automatically `./runner.sh -t <path to .ipynb file>`

### CODE REVIEW

**Comment by mingxin-zheng** [comment]
The new `build / copyright ` action fails because lots of files miss copyright info, and the action plan is mentioned in #1143

File: runner.sh [hunk 1]
```diff
@@ -116,6 +119,7 @@ function print_usage {
     echo "    --no-run          : don't run notebooks"
     echo "    --no-checks       : don't run code checks"
     echo "    --autofix         : autofix where possible"
+	echo "    --copyright       : check whether every source code and notebook has a copyright header"
```
Comment (wyli) [inline runner.sh:122]: there are some indentation misalignments in the script

File: runner.sh [hunk 2]
```diff
@@ -227,6 +235,58 @@ if [ $doChecks = true ]; then
...
+    celltype=$4
+    ${PY_EXE} - << END
+import nbformat
+import re
+
+with open("$fname", "r") as f:
+    contents = nbformat.reads(f.read(), as_version=4)
+    cell = contents.cells[int("$cell")]
+    result = "true" if cell.cell_type == "$celltype" and re.search("$key", cell.source) else "false"
+print(result)
```
Comment (wyli) [inline runner.sh:251]: this `print` doesn't seem to print anything?

**Review by wyli — APPROVED** [review]
thanks, it loos good to me, feel free to revise or ignore my comments

File: runner.sh (hunk 1 above)
Comment (mingxin-zheng) [reply runner.sh]: Fixed the issue to make the indentation consistent in the function, plus a few other places in my previous commit. Thanks @wyli !

File: runner.sh (hunk 2 above)
Comment (mingxin-zheng) [reply runner.sh:251]: It prints string `"true"` or `"false"`, and the message in the stdout will be consumed by the following line:

```sh
if [[ $(verify_notebook_has_key_in_cell "$fname" "$license" 0 "markdown" ) != true ]];
```

If there are better ways to handle it, please let me know. Thanks @wyli !

Comment (wyli) [reply runner.sh:251]: I see, thanks! I think it'll be more readable to have a separate python script and a bash variable to store the returned value, like this example https://github.com/Project-MONAI/model-zoo/blob/3bffe793f13a80aa648b559d3f529bb701792ac6/ci/run_regular_tests_cpu.sh#L56-L57 but the current implementation is also ok

**Comment by udiram** [comment]
@mingxin-zheng I'm seeing this problem with my PR (#1129), where it checks every file for the copyright in CI/CD, and fails. do we have to wait until all existing files are updated to include it to merge the PR?

File: runner.sh (hunk 2 above)
Comment (mingxin-zheng) [reply runner.sh:253]: Thanks @wyli . The suggested way is implemented in #1152 .

**Pull request merged**